pkg/apis/fleet.cattle.io/v1alpha1/bundle.go

```diff
@@ -282,6 +283,7 @@ type BundleDeploymentStatus struct {
 	ModifiedStatus      []ModifiedStatus                    `json:"modifiedStatus,omitempty"`
 	Display             BundleDeploymentDisplay             `json:"display,omitempty"`
 	SyncGeneration      *int64                              `json:"syncGeneration,omitempty"`
+	ReapplyAfter        string                              `json:"reapplyAfter,omitempty"`
```

Comment:
What is this used for? To prevent the bundle being reapplied in 5 minutes?

Reply:
You can use metav1.Time to avoid parsing... and also the best pratice is to record the last time it was reapplied and check if lastAppliedTime is within 5 minutes...

Reply:
yeah the field is to record the last time it was applied. I make the changes.